app/views/Tagging/index.tsx

```diff
@@ -38,6 +58,20 @@ function Tagging(props: Props) {
     const { user } = useContext(UserContext);
     const { project } = useContext(ProjectContext);
 
+    const variables = useMemo(() => (
+        project?.id ? { projectId: project.id } : undefined
+    ), [project]);
+
+    const {
+        data,
+    } = useQuery<ConnectorSourcesCountQuery, ConnectorSourcesCountQueryVariables>(
+        CONNECTOR_SOURCES_COUNT,
+        {
+            skip: !variables,
+            variables,
+        },
+    );
```

Comment:
Don't we need to refresh this after some sources are added from connectors?

Reply:
I think its fine.

Reply:
i do already